CHANGELOG.md

```diff
@@ -11,6 +11,8 @@ This beta release includes a feature with a known issue: the `within` expression
  - Added methods to set and get the sort key of features in a `FillLayer` and `LineLayer` at runtime. Features with a higher sort key will appear above features with a lower sort key. ([#209](https://github.com/mapbox/mapbox-gl-native-android/pull/209))
  - Introduced option to set the minimum and maximum pitch of a map. ([#199](https://github.com/mapbox/mapbox-gl-native-android/pull/199))
  - Introduced ability to configure the zoom level(s) that tiles are prefetched from by `Source`. This overrides the  tile prefetch setting defined by the `Map` instance. ([#184](https://github.com/mapbox/mapbox-gl-native-android/pull/184))
+ - Added new API to add runtime layers/sources/images to MapSnapshotter . ([#268](https://github.com/mapbox/mapbox-gl-native-android/pull/268))
```

Comment:
This is a big feature and should be higher in the changelog! :) 

@tobrun @Chaoba @alexshalamov are there any other improvements to MapSnapshotter to call out besides the fact that you can now (1) manipulate the layers and sources of a style used by the snapshotter and (2) use runtime style property APIs on the snapshotter?

If so, I would be happy to call out the snapshotter in a separate section like @mapbox/maps-ios did: https://github.com/mapbox/mapbox-gl-native-ios/blob/master/platform/ios/CHANGELOG.md

If we made other improvements to the snapshotter that would make it more stable or thread-safe, etc., we should call that out, so let me know if there are additional details that we could include.

Reply:
@chloekraw afsk, there is no other more breaking changes. I will update the change log later to call out snapshotter in a separate section.

Reply:
Ok! No problem. If there are no other changes, we should combine both entries into a single one and move the bullet under the new expressions we launched